mobify/platform-scaffold
# Use package.json's siteUrl

There are a couple places where we've hardcoded a URL:
* The preview links from running `npm run dev`
* The `site.js` file that configures environments for end-to-end tests

## Changes
- Instead of a hardcoded URL, use the value from `package.json`'s 'siteUrl' 

## How to test-drive this PR
- Checkout this branch
- Change the value for `siteUrl` in `package.json`
- `npm run dev` and observe that the Preview link has the site URL set to your given URL
- Or `npm run smoke-test` and observe that the Preview page has the site URL set to your given URL

<img width="642" alt="screen shot 2017-03-20 at 3 43 52 pm" src="https://cloud.githubusercontent.com/assets/7492180/24125100/11dcf5f6-0d84-11e7-8077-e06ec219cf31.png">
